homeassistant/components/roborock/sensor.py

```diff
@@ -287,7 +283,7 @@ def _dock_error_value_fn(state: DeviceState) -> str | None:
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
     RoborockSensorDescriptionA01(
-        key="error",
+        key="dyad_error",
```

Comment:
i believe that changing this key will change unique ids which will break existing use cases. can this be reverted?